week3_model_free/qlearning.ipynb

```diff
@@ -285,11 +285,11 @@
    "source": [
     "# Binarized state spaces\n",
     "\n",
-    "Use agent to train efficiently on `CartPole-v0`. This environment has a continuous set of possible states, so you will have to group them into bins somehow.\n",
+    "Use an agent to train efficiently on `CartPole-v0`. This environment has a continuous set of possible states, so you will have to group them somwhow into bins.\n",
     "\n",
-    "The simplest way is to use `round(x, n_digits)` (or `np.round`) to round a real number to a given amount of digits. The tricky part is to get the `n_digits` right for each state to train effectively.\n",
+    "The simplest way is to use `round(x, n_digits)` (or `np.round`) to round the real number to a given amount of digits. The tricky part is to get the `n_digits` right for each state to train effectively.\n",
```

Comment:
I think "a real number" is fine here, since there is no discussion of real numbers before this point.